ndcube/ndcube.py

```diff
@@ -886,28 +886,32 @@ def _new_instance_from_op(self, new_data, new_unit, new_uncertainty):
     def __array_ufunc__(self, ufunc, method, *inputs, **kwargs):
         if method == '__call__':
             if ufunc == np.add:
-                if isinstance(inputs[0], NDCube):
+                if isinstance(inputs[0], NDCube) and kwargs.get("dunder"):
                     new_data = inputs[0].data + inputs[1]
                     return new_data
```

Comment:
Does this mean that `np.add(cube1, array2)` would return an array?

This looks different to the implementation for `np.sub` which appears like it would return an `NDCube`.

Reply:
`np.add(cube1, array2)` would return an NDCube.
The `np.subtract` utilises the dunder method `__sub__` and in turn the __sub__ dunder method uses the `__add__` dunder method and since `np.subtract` utilises the `__add__` so we don't need to check the way we did in `np.add` as `__add__` dunder method will call the `np.add` which in turn will do the checking and hence both returns the same type.